package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "frontend-worship-decisions",
-  "version": "0.11.0",
+  "version": "0.12.0-rc.1",
```

Comment:
I would roll this commit back, since you used a `feature/*` prefix for the branch name, [it already built an image you can use](https://build.redpencil.io/repos/1321/pipeline/107/3). I would use that instead. Tagging versions in PRs isn't a good practice IMO and should be avoided.

Reply:
Check, I've used that only for convenience but I'll avoid doing it.

Reply:
Yea, I understand. It's just that the "feature" image builds fill the same roll and you don't have to create tags that are potentially incorrect (for example, if this PR wasn't merged for some reason).